python/tests/core/metrics/test_unicode_range.py

```diff
@@ -26,6 +26,19 @@ def test_unicode_range_metric() -> None:
     assert metric.submetrics[_STRING_LENGTH].mean.value == np.array([len(s) for s in strings]).mean()
 
 
+def test_unicode_range_metric_upper_case() -> None:
+    config = MetricConfig(unicode_ranges={"lower": (97, 122), "upper": (65, 90)}, lower_case=False)
```

Comment:
Could we put in a comment what these lower and uppers mean? I always forget them off the top of my head. It would be nice for readability